app/services/common_indexers/image.rb

```diff
@@ -73,6 +73,10 @@ def fields
 
     private
 
+      def nul_collections
+        member_of_collections.select { |c| c.collection_type_gid == Settings.nul_collection_type }.map { |c| { id: c.id, title: c.title.to_a } }
```

Comment:
```suggestion
        member_of_collections.select { |c| Hyrax::CollectionType.find_by(machine_id: Hyrax::CollectionType::USER_COLLECTION_MACHINE_ID).collections.include?(c) }.map { |c| { id: c.id, title: c.title.to_a } }
```

Reply:
I don't think that will work. We're trying to only add collection information if it's an "NUL Collection" (or "NUL Collections" on staging) type collection - which is a manually created collection type. - it's not a user collection type.

Reply:
Sorry I misunderstood the feature. I think you could do something like `Hyrax::CollectionType.find_by(title: "NUL Collection")` or the like instead.